### ISSUE TITLE
Consider more images for FID

### ISSUE THREAD

**Issue by mailvijayasingh** [ahmadki/mlperf_sd_inference#1]
This is more like a discussion, shall we consider more generated images per prompt  while calculating FID? The generated images can be very different if the seed changes, shall we do some aggregation for FID? Trying to understand the implication of considering one generated image per prompt.

**Comment by ahmadki** [comment]
This is a valid point, in the spreadsheet I shared there are three runs for each experiment to measure the run to run variance. Past a certain number of samples, the variance is low, so generating multiple images per prompt shouldn't change the final score. 

We'll discuss this point in more details in the next meeting.

**Comment by mailvijayasingh** [comment]
Yes, lets discuss. Also, I still don't have access to the sheet.
